template/component-top-nav.html.jinja

```diff
@@ -1,15 +1,15 @@
-{# <div class="top-banner fixed-top"> 
+<div class="top-banner fixed-top"> 
   <!-- mobile -->
-  <div class="justify-content-center d-md-none">
+  <!-- <div class="justify-content-center d-md-none">
     <span><p class="mb-0">{% trans %}Registration is live for Apex: The XRPL{% endtrans %}</p></span>
-    <span><p class="mb-0"> Developer Summit <a href="https://www.apexdevsummit.com/" target="_blank" class="btn btn-outline-secondary">Register Now</a></p></span>
-  </div>
+    <span><p class="mb-0"> Developer Summit <a href="https://www.apexdevsummit.com/" target="_blank" class="btn btn-outline-secondary">Start Learning</a></p></span>
+  </div> -->
```

Comment:
Rather than commenting out the old banner, can we fully delete it?

Reply:
Removed.